nixos/modules/services/misc/linux-enable-ir-emitter.nix

```diff
@@ -0,0 +1,47 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
+  cfg = config.services.linux-enable-ir-emitter;
+in
+{
+  options = {
+    services.linux-enable-ir-emitter = {
+      enable = mkEnableOption (mdDoc "Linux Enable IR Emitter");
+
+      package = mkPackageOptionMD pkgs "linux-enable-ir-emitter" {} // {
+        description = mdDoc ''
+          Package to use for the Linux Enable IR Emitter service.
+        '';
+      };
+
+      device = mkOption {
+        type = types.lines;
+        default = "video2";
```

Comment:
(1) types.lines merges/joins multiple definitions -- shouldn't this be types.str (only one value allowed). (2) shouldn't the default be `/dev/...`? Or add an example showing fully working value?

Reply:
Yes, I should make the option more explicit. I'll make it take `types.str` and have a default value of `/dev/video2`. It's supposed to be the IR camera that will be used for driver configuring the blinking mode.

Reply:
I still left it `video2` but added an explanation, because only the part after `/dev/` is used due by the systemd device.